modules/workflows/OrcaCopyToGlacierWorkflow/orca_copy_to_glacier_workflow.asl.json

```diff
@@ -50,7 +50,9 @@
             "multipart_chunksize_mb": "{$.meta.collection.meta.multipart_chunksize_mb}",
             "excludeFileTypes": "{$.meta.collection.meta.excludeFileTypes}",
             "providerId": "{$.meta.provider.id}",
-            "executionId": "{$.cumulus_meta.execution_name}"
+            "executionId": "{$.cumulus_meta.execution_name}",
+            "collection_shortname": "{$.meta.collection.name}",

```

Comment:
We have a mixture of snake case and camel case here. Since Cumulus uses Camelcase for this type of configuration, I am wondering if we shouldn't do the same here maybe `collectionShortname` and `collectionVersion`. Thoughts?

Reply:
makes sense. Changed